src/rdpq/rdpq_xform.c

```diff
@@ -0,0 +1,364 @@
+/**
+ * @file rdpq_text.c
+ */
+#include "fgeom2d.h"
+#include "rdpq_xform_internal.h"
+#include "rdpq_xform.h"
+#include "rdpq_rect.h"
+#include "rdpq_tri.h"
+
+#include "debug.h"
+#include "utils.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdarg.h>
+
+//List of transformation types
+//This is ordered from most strict to least strict
+typedef enum {
```

Comment:
Please us doxygen format here. Also maybe explain what "strict" means in this context.

Reply:
Documentation changed to use doxygen format and to use alternative phrasing.